src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,44 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconProps } from "@components/Icon/Icon";
+import { Dispatch, ReactElement, SetStateAction } from "react";
+import css from "./Slider.module.css";
+
+export interface IconItem {
+    icon: ReactElement<IconProps>;
+    name: string;
+}
+
+export interface SliderProps {
+    items: string[] | IconItem[];
```

Comment:
IMO it would be better to use an object with an `id` prop instead of plain string. atm if the string is translated the dev who uses the `Slider` somehow needs to map the selected string to a unique item outside of the `Slider`. do you know what i mean? 🙂

Reply:
yes, good idea!